api/envoy/api/v2/auth/cert.proto

```diff
@@ -433,13 +433,16 @@ message DownstreamTlsContext {
 
     // Config for fetching TLS session ticket keys via SDS API.
     SdsSecretConfig session_ticket_keys_sds_secret_config = 5;
+
+    // Config to disable TLS session tickets altogether
+    bool disable_session_tickets = 6;
   }
 
   // If specified, session_timeout will change maximum lifetime (in seconds) of TLS session
   // Currently this value is used as a hint to `TLS session ticket lifetime (for TLSv1.2)
   // <https://tools.ietf.org/html/rfc5077#section-5.6>`
   // only seconds could be specified (fractional seconds are going to be ignored).
-  google.protobuf.Duration session_timeout = 6 [(validate.rules).duration = {
+  google.protobuf.Duration session_timeout = 7 [(validate.rules).duration = {
```

Comment:
This can't be changed and is a breaking change. The field above should be numbered 7. cc @htuch we should probably figure out protolock or something like that to make sure CI would catch this in the future...

Reply:
Totally, tracking issue at https://github.com/envoyproxy/envoy/issues/3368, we need to triage it with the rest of the API work.

Reply:
Just a friendly ping here, please update this prior to merge.

Reply:
Addressed API feedback in https://github.com/envoyproxy/envoy/pull/10178/commits/7e35a69f32727e405016579dd9076ae4df239e1b